packages/react-router/lib/dom/ssr/errorBoundaries.tsx

```diff
@@ -83,7 +84,7 @@ export function RemixRootDefaultErrorBoundary({
       dangerouslySetInnerHTML={{
         __html: `
         console.log(
-          "💿 Hey developer 👋. You can provide a way better UX than this when your app throws errors. Check out https://remix.run/guides/errors for more information."
+          "💿 Hey developer 👋. You can provide a way better UX than this when your app throws errors. Check out https://reactrouter.com/how-to/error-boundary for more information."
```

Comment:
I think it would be better to keep links pointing towards Remix docs instead of RR

CC/ @brookslybrand @brophdawg11 

```suggestion
          "💿 Hey developer 👋. You can provide a way better UX than this when your app throws errors. Check out https://remix.run/guides/errors for more information."
```